ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Substra Repository
+  - url: https://github.com/SubstraFoundation/substra-documentation
```

Comment:
I think there is a syntax issue here (see https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser)

It should be
``` yaml
contact_links:
  - name: Substra Repository
    url: https://github.com/SubstraFoundation/substra-documentation
```

Also, the documentation isn't a place to report issues, I'm not sure we should add a contact link pointing to it.

Reply:
Thank you for this check @jmorel !

Reply:
Do you think that it would be better to point towards the main substra repository?

Reply:
It looks like contact links are made to direct people outside of github, according to the documentation:

> If you prefer to receive certain reports outside of GitHub, you can direct people to external sites with contact_links.

These contact links are especially relevant if you disable blank issues, but since we allow them we could remove the contact links entirely.